modules/serverless-workflow-tag-revisions-assign.adoc

```diff
@@ -0,0 +1,44 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="assigning-tag-revisions_{context}"]
+= Assigning tag revisions 
+
+A tag in a traffic block of service creates a custom URL, which points to a referenced revision. A user can define a unique tag for an available revision of a service which creates a custom URL by using the format `http(s)://TAG-SERVICE.DOMAIN`.
+
+A given tag must be unique to its traffic block of the service. `kn` supports assigning and unassigning custom tags for revisions of services as part of the `kn service update` command.
+
+[NOTE]
+====
+If you have assigned a tag to a particular revision, a user can reference the revision by its tag in the `--traffic` flag as `--traffic Tag=Percent`.
+====
+
+
+.Procedure
+* Use the following command:
+
```

Comment:
Put a "+" on this line so that the following command is indented properly with it.